disco_aws_automation/disco_acm.py

```diff
@@ -6,27 +6,45 @@
 import boto3
 import botocore
 
+CERT_SUMMARY_LIST_KEY = 'CertificateSummaryList'
+CERT_ARN_KEY = 'CertificateArn'
+DOMAIN_NAME_KEY = 'DomainName'
+
 
 class DiscoACM(object):
     """
     A class to manage the Amazon Certificate Service
 
     """
+    WILDCARD_PREFIX = "*."
 
     def __init__(self, connection=None):
         self._acm = connection
 
+    def _in_domain(self, domain, dns_name):
```

Comment:
@kelibezhani sorry I wasn't clear initially.  In the case `dns_name` is **a.b.c** and `domain` is **\*.a**, after the `if` `domain` would be **a**.  Wouldn't `domain in dns_name` incorrectly return True?

@pchand-amplify that's true, but I don't want to get caught up in too much validation.